src/wp-content/themes/twentysixteen/inc/template-tags.php

```diff
@@ -19,6 +19,13 @@
 	 */
 	function twentysixteen_entry_meta() {
 		if ( 'post' === get_post_type() ) {
+			/**
+			 * Filters the Twenty Sixteen author avatar size.
+			 *
+			 * @since Twenty Sixteen 1.0
+			 *
+			 * @param int The height and width avatar dimension in pixels. Default 49.
```

Comment:
Twenty Sixteen has two different filterable avatar sizes for post authors. This docblock could be similar to the one for [`twentysixteen_author_bio_avatar_size`](https://github.com/WordPress/wordpress-develop/blob/6.8.1/src/wp-content/themes/twentysixteen/template-parts/biography.php#L15-L22), but hopefully the description helps clarify the difference.

```suggestion
			 * Filters the Twenty Sixteen entry meta avatar size.
			 *
			 * @since Twenty Sixteen 1.0
			 *
			 * @param int $size The avatar height and width size in pixels.
```